core/tests/protractor_utils/ProfilePage.js

```diff
@@ -41,18 +41,26 @@ var ProfilePage = function() {
   };
 
   this.expectCurrUserToHaveProfilePhoto = async function() {
+    await waitFor.visibilityOf(currUserProfilePhoto,
+      'Current user profile picture label element taking too long to appear');
     expect(await currUserProfilePhoto.isPresent()).toBe(true);
   };
 
   this.expectOtherUserToHaveProfilePhoto = async function() {
+    await waitFor.visibilityOf(otherUserProfilePhoto,
+      'otherUserProfilePhoto label element taking too long to appear');
     expect(await otherUserProfilePhoto.isPresent()).toBe(true);
   };
 
   this.expectUserToHaveBio = async function(expectedText) {
+    await waitFor.visibilityOf(bio,
+      'bio label element taking too long to appear');
     expect(await bio.getText()).toMatch(expectedText);
   };
 
   this.expectUserToHaveNoInterests = async function() {
+    await waitFor.visibilityOf(interests,
+      'interests label element taking too long to appear'); //Err
```

Comment:
I don't think this will work. Since we expect there to be no interests present, the `waitFor` here will fail. Let me know if I'm wrong though!

Also, the `//Err` comment looks like it should be removed?